ios/brave-ios/Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/Paged/RequestBlockingScript.js

```diff
@@ -17,6 +26,8 @@ window.__firefox__.execute(function($) {
         resourceType: 'xmlhttprequest'
       }
     }).then(blocked => {
+      blockingCache.set(resourceURL.href, blocked)
```

Comment:
and what do we do with the cache ?

Reply:
The cache stays there but its wiped on every page reload